runners/spark/src/test/java/org/apache/beam/runners/spark/translation/SparkExecutableStageFunctionTest.java

```diff
@@ -101,7 +104,9 @@ public void setUpMocks() throws Exception {
   public void sdkErrorsSurfaceOnClose() throws Exception {
     SparkExecutableStageFunction<Integer, ?> function = getFunction(Collections.emptyMap());
     doThrow(new Exception()).when(remoteBundle).close();
-    function.call(Collections.emptyIterator());
+    List<WindowedValue<Integer>> dummyList = new ArrayList<>();
```

Comment:
Nit: "dummy" sounds a lot like mocks or stubs, which are technical terms that describe the implementation of test objects. This list isn't a "dummy" in that sense; it's a real list, albeit one constructed for test purposes.

tl;dr rename this `inputs`.